packages/discovery-react-components/src/utils/document/__tests__/transformEnrichment.spec.tsx

```diff
@@ -0,0 +1,82 @@
+import transformEnrichment from '../transformEnrichment';
+import isEqual from 'lodash/isEqual';
+import contractData from 'components/CIDocument/components/CIDocument/__fixtures__/contract.json';
+import invoiceData from 'components/CIDocument/components/CIDocument/__fixtures__/invoice-index_op.json';
+import purchaseOrderData from 'components/CIDocument/components/CIDocument/__fixtures__/po-index_op.json';
+import { EnrichedHtml } from '../../../components/CIDocument/types';
```

Comment:
this can just be
```suggestion
import { EnrichedHtml } from 'components/CIDocument/types';
```

Reply:
For some reason, VSCode **was** giving error for the other import, now it is not 😄